cmd/nginx-ingress/main.go

```diff
@@ -241,70 +256,72 @@ func createConfigAndKubeClient() (*rest.Config, *kubernetes.Clientset) {
 					Server: *proxyURL,
 				},
 			}).ClientConfig()
-		if err != nil {
-			glog.Fatalf("error creating client configuration: %v", err)
-		}
 	} else {
-		if config, err = rest.InClusterConfig(); err != nil {
-			glog.Fatalf("error creating client configuration: %v", err)
-		}
+		config, err = rest.InClusterConfig()
 	}
 
-	kubeClient, err := kubernetes.NewForConfig(config)
-	if err != nil {
-		glog.Fatalf("Failed to create client: %v.", err)
-	}
+	return config, err
+}
 
-	return config, kubeClient
+func getKubeClient(config *rest.Config) (kubeClient *kubernetes.Clientset, err error) {
+	kubeClient, err = kubernetes.NewForConfig(config)
+	return kubeClient, err
 }
 
-func kubernetesVersionInfo(kubeClient kubernetes.Interface) {
+func confirmMinimumkubernetesVersionCriteria(kubeClient kubernetes.Interface) (err error) {
```

Comment:
capitalise the K in the function name

Reply:
Sorted with a further rename to `confirmMinimumK8sVersionCriteria`.